src/luxos/exceptions.py

```diff
@@ -0,0 +1,24 @@
+class LuxosBaseException(Exception):
+    pass
+
+
+class MinerConnectionError(LuxosBaseException):
+    def __init__(self, host: str, port: int, *args, **kwargs):
+        super().__init__(host, port, *args, **kwargs)
+        self.address = (host, port)
+
+    def __str__(self):
+        return (f"<{self.address[0]}:{self.address[1]}>: {self.__class__.__name__}, "
+                f"{self.args[2] if self.args[2:] else 'unknown reason'}")
+
+
+class MinerCommandSessionAlreadyActive(MinerConnectionError):
```

Comment:
I assume those are still todo so all good that we already have them in place here.

Reply:
Yes, so far I wrapped the one used in the `luxos.asyncops.wrapped` functions, to each exception caiies over the host/port informations over.